runners/wikiConsole/templates/downloadFile.asciidoc

```diff
@@ -0,0 +1,32 @@
+Download a File
+===============
+
+Download the file in a Terminal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+First, you need to open a command prompt in your current Workspace. For Windows, you can use Powershell and on Linux, you can use the Terminal.
+
+Prerequisites
+~~~~~~~~~~~~~
+The Folder you want to load the file has to be existing. If the Folder doesn't exist you can create it from Terminal.
+
+.OS
```

Comment:
Is the '.OS' here needed?

Reply:
I find it easier to read and the structure makes it easier to skip over the part that is not required for you.